spec/index.adoc

```diff
@@ -1623,6 +1623,8 @@ The return result is an extended public key consisting of an ECDSA `public_key`,
 This call requires that the ECDSA feature is enabled, and the `canister_id` meets the requirement of a canister id.
```

Comment:
Not blocking this change, but for a follow-up PR:

Not sure if it makes sense to mention that the `ECDSA` feature needs to be enabled. It doesn't need to be enabled on the calling canister's subnet anyway - only on some subnet holding the key. But if it wasn't enabled on any subnet holding the key then the key wouldn't even exist...

Maybe we should say something like "the key_id must exist on some IC subnet".